frontend/public/components/deployment-config.tsx

```diff
@@ -103,70 +106,82 @@ export const menuActions: KebabAction[] = [
 ];
 
 export const DeploymentConfigDetailsList = ({ dc }) => {
+  const { t } = useTranslation();
   const timeout = _.get(dc, 'spec.strategy.rollingParams.timeoutSeconds');
   const updatePeriod = _.get(dc, 'spec.strategy.rollingParams.updatePeriodSeconds');
   const interval = _.get(dc, 'spec.strategy.rollingParams.intervalSeconds');
   const triggers = _.map(dc.spec.triggers, 'type').join(', ');
   return (
     <dl className="co-m-pane__details">
-      <DetailsItem label="Latest Version" obj={dc} path="status.latestVersion" />
-      <DetailsItem label="Message" obj={dc} path="status.details.message" hideEmpty />
-      <DetailsItem label="Update Strategy" obj={dc} path="spec.strategy.type" />
+      <DetailsItem label={t('workload~Latest version')} obj={dc} path="status.latestVersion" />
+      <DetailsItem label={t('workload~Message')} obj={dc} path="status.details.message" hideEmpty />
+      <DetailsItem label={t('workload~Update strategy')} obj={dc} path="spec.strategy.type" />
       {dc.spec.strategy.type === 'RollingUpdate' && (
         <>
           <DetailsItem
-            label="Timeout"
+            label={t('workload~Timeout')}
             obj={dc}
             path="spec.strategy.rollingParams.timeoutSeconds"
             hideEmpty
           >
-            {pluralize(timeout, 'second')}
+            {t('workload~second', { count: timeout })}
           </DetailsItem>
           <DetailsItem
-            label="Update Period"
+            label={t('workload~Update period')}
             obj={dc}
             path="spec.strategy.rollingParams.updatePeriodSeconds"
             hideEmpty
           >
-            {pluralize(updatePeriod, 'second')}
+            {t('workload~second', { count: updatePeriod })}
           </DetailsItem>
           <DetailsItem
-            label="Interval"
+            label={t('workload~Interval')}
             obj={dc}
             path="spec.strategy.rollingParams.intervalSeconds"
             hideEmpty
           >
-            {pluralize(interval, 'second')}
+            {t('workload~second', { count: interval })}
           </DetailsItem>
           <DetailsItem
-            label="Max Unavailable"
+            label={t('workload~Max unavailable')}
             obj={dc}
             path="spec.strategy.rollingParams.maxUnavailable"
           >
-            {dc.spec.strategy.rollingUpdate.maxUnavailable ?? 1} of{' '}
-            {pluralize(dc.spec.replicas, 'pod')}
+            {t('workload~{{maxUnavailable}} of pod', {
+              maxUnavailable: dc.spec.strategy.rollingUpdate.maxUnavailable ?? 1,
+              count: dc.spec.replicas,
+            })}
```

Comment:
Does this display correctly? I think you need `{{count}}` in the message here?

```suggestion
            {t('workload~{{maxUnavailable}} of {{count}} pod', {
              maxUnavailable: dc.spec.strategy.rollingUpdate.maxUnavailable ?? 1,
              count: dc.spec.replicas,
            })}
```

Reply:
fixed - same problem in deployments